packages/components/src/palette-edit/stories/index.story.tsx

```diff
@@ -56,6 +56,7 @@ Default.args = {
 	colors: [
 		{ color: '#1a4548', name: 'Primary', slug: 'primary' },
 		{ color: '#0000ff', name: 'Secondary', slug: 'secondary' },
+		{ color: '#fb326b', name: 'Tertiary', slug: 'tertiary' },
```

Comment:
Increased this to three colors so we can better check the border radiuses in details view.